server/game/player.js

```diff
@@ -921,7 +921,7 @@ class Player extends GameObject {
             holding: dynastyCardLocations,
             conflictCharacter: [...conflictCardLocations, Locations.PlayArea],
             dynastyCharacter: [...dynastyCardLocations, Locations.PlayArea],
-            event: [...conflictCardLocations, Locations.BeingPlayed],
+            event: _.uniq([...conflictCardLocations, ...dynastyCardLocations, Locations.BeingPlayed]),
```

Comment:
Underscore D:<

Reply:
Looks like the alternative is pretty verbose, so unless we're looking to completely remove underscore dependencies this is the best approach.